llvm/tools/llvm-cas/llvm-cas.cpp

```diff
@@ -592,27 +627,27 @@ static ObjectRef importNode(ObjectStore &CAS, ObjectStore &UpstreamCAS,
 }
 
 static int import(ObjectStore &CAS, ObjectStore &UpstreamCAS,
-                  ArrayRef<std::string> Objects) {
+                  ArrayRef<std::string> InputFiles) {
   ExitOnError ExitOnErr("llvm-cas: import: ");
 
-  for (StringRef Object : Objects) {
+  for (StringRef Object : InputFiles) {
     CASID ID = ExitOnErr(CAS.parseID(Object));
     importNode(CAS, UpstreamCAS, ID);
   }
   return 0;
 }
 
 static int putCacheKey(ObjectStore &CAS, ActionCache &AC,
-                       ArrayRef<std::string> Objects) {
+                       ArrayRef<std::string> InputFiles) {
```

Comment:
Don't rename those function parameters. They are actually `CASObject`, not input files. Same for lots of other function parameter.